spec/index.md

```diff
@@ -1954,6 +1954,15 @@ The optional `settings` parameter can be used to set the following settings:
 
     Default value: 5_000_000_000_000 (5 trillion cycles).
 
+-   `wasm_memory_limit` (`nat`)
+
+    Must be a number between 0 and 2<sup>48</sup>-1 (i.e., 256TB), inclusively, and indicates the upper limit on the WASM heap memory consumption of the canister.
```

Comment:
Is the range validated somewhere in the code?

Reply:
Good catch! The implementation currently allows 2^64. I followed the memory_allocation validation, but that enforces the 2^48 implicitly via the subnet available memory.

Reply:
I'll fix it.